### ISSUE TITLE
m3u video format support for asbplayer website

### ISSUE THREAD

**Issue by miroshQa** [killergerbah/asbplayer#569]
There is such a wonderful program as [Tor Server](https://github.com/YouROK/TorrServer); it allows you to paste a torrent link, then press the "playlist" button. After that, it gives you an [.m3u file](https://en.wikipedia.org/wiki/M3U) that you can watch in a video player. It works in [VLC](https://www.videolan.org/vlc/), and I really want to be able to do the same on the [asbplayer website](https://killergerbah.github.io/asbplayer) because now it says what it is format is not supported

![image](https://github.com/user-attachments/assets/d24a7724-0b65-47f4-b290-d7376a8b9d38)

![image](https://github.com/user-attachments/assets/a636d4db-2690-4480-8fc4-1d637dedbfc3)

![image](https://github.com/user-attachments/assets/990d22e9-ed86-465b-8e53-46385fa6adb8)

**Comment by killergerbah** [comment]
@miroshQa Why did you close the issue?

**Comment by miroshQa** [comment]
> @miroshQa Why did you close the issue?

Sorry, it was bad of me to close the issue without comments

I closed this issue because I found more problems with this player than the lack of m3u support. 

Specifically, I can”t select an audio track and also don”t have sound when playing a local video. 

As a result, I launched the video locally in Jellyfin and all the listed problems were solved there

And as for the m3u, I realized that uploading videos before watching is quite a normal option.
